### PR TITLE
create popup text; remove initial text implementation

### PR DESCRIPTION
**Pull request by emshuttles**

### CODE REVIEW

File: Assets/Scripts/Combat/CreatureStats.cs [hunk 1]
```diff
@@ -147,8 +135,7 @@ private int BonusRearDamage()
 
     private void DisplayPopup(string text)
     {
-        textLabelTimer = 5.0f;
-        textLabelText = text;
+        PopupTextController.CreatePopupText(text, transform);
```
Comment (dspages) [inline Assets/Scripts/Combat/CreatureStats.cs:138]: Might look better if the popup texts are a little less verbose (at MINIMUM, get rid of "inflicted").

File: Assets/Scripts/Combat/PopupTextController.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    public static void Initialize()
+    {
+        canvas = GameObject.Find(canvasName);
+        popupTextPrefab = AssetDatabase.LoadAssetAtPath<PopupText>(popupTextPrefabFilePath);
+    }
+
+    public static void CreatePopupText(string text, Transform transform)
+    {
+        PopupText instance = Instantiate(popupTextPrefab);
+        Vector2 screenPosition = Camera.main.WorldToScreenPoint(transform.position);
```
Comment (dspages) [inline Assets/Scripts/Combat/PopupTextController.cs:20]: Some games have the popup text appear at an elevated Y coordinate relative to the unit - up to you if you think it looks better that way or not.

**Review by dspages — APPROVED** [review]
Looks good! Only had a couple of minor aesthetic comments.

File: Assets/Scripts/Combat/PopupTextController.cs (hunk 2 above)
Comment (emshuttles) [reply Assets/Scripts/Combat/PopupTextController.cs:20]: I considered that, but all the units have status bars above their heads so I wasn't sure where to put the text. It'll be easy to change later once there's a firmer direction for the UI.

File: Assets/Scripts/Combat/CreatureStats.cs (hunk 1 above)
Comment (emshuttles) [reply Assets/Scripts/Combat/CreatureStats.cs:138]: Done

**Pull request merged**